plugins/attachment-resources/src/components/FileBrowser.svelte

```diff
@@ -282,11 +284,17 @@
           <div class="item flex">
             <AttachmentPresenter value={attachment} />
           </div>
-          <div class="eAttachmentRowActions" class:fixed={i === selectedFileNumber}>
-            <div id="context-menu" class="eAttachmentRowMenu" on:click={(event) => showFileMenu(event, attachment, i)}>
-              <IconMoreV size={'small'} />
+          {#if attachment.modifiedBy !== myAccId}
```

Comment:
Menu can contains other actions, you should hide action, not menu

Reply:
Done